### ISSUE TITLE
Build script fails on Raspi 3 with Debian 12 bookworm

### ISSUE THREAD

**Issue by zetric** [knxd/knxd#608]
Debian build fails at step 3
```
$ sh install-debian.sh
...
dh_missing: warning: lib/systemd/system/knxd.service exists in debian/tmp but is not installed to anywhere
dh_missing: warning: lib/systemd/system/knxd.socket exists in debian/tmp but is not installed to anywhere
dh_missing: warning: usr/bin/findknxusb exists in debian/tmp but is not installed to anywhere
dh_missing: warning: usr/lib/aarch64-linux-gnu/libeibclient.a exists in debian/tmp but is not installed to anywhere
dh_missing: warning: usr/lib/aarch64-linux-gnu/libeibclient.la exists in debian/tmp but is not installed to anywhere
        The following debhelper tools have reported what they installed (with files per package)
         * dh_install: knxd (5), knxd-dev (12), knxd-tools (4)
         * dh_installdocs: knxd (3), knxd-dev (0), knxd-tools (0)
        If the missing files are installed by another tool, please file a bug against it.
        When filing the report, if the tool is not part of debhelper itself, please reference the
        "Logging helpers and dh_missing" section from the "PROGRAMMING" guide for debhelper (10.6.3+).
          (in the debhelper package: /usr/share/doc/debhelper/PROGRAMMING.gz)
        Be sure to test with dpkg-buildpackage -A/-B as the results may vary when only a subset is built
        If the omission is intentional or no other helper can take care of this consider adding the
        paths to debian/not-installed.

        Remember to be careful with paths containing "aarch64-linux-gnu", where you might need to
        use a wildcard or (assuming compat 13+) e.g. ${DEB_HOST_MULTIARCH} in debian/not-installed
        to ensure it works on all architectures (see #961104).
dh_missing: error: missing files, aborting
make: *** [debian/rules:13: binary] Error 255
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
```
See full output here: [https://pastebin.com/nrJudPzQ](https://pastebin.com/nrJudPzQ)

Used commit: e779c955556ee4f5c02a4d15abc70ec84ae6d82c

```
$ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

$ uname -a
Linux emkhab 6.6.62+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.62-1+rpt1 (2024-11-25) aarch64 GNU/Linux
```
Raspberry Pi model 3, Openhabian image based on RaspiOS base image November 2024

Hints in build description are checked (it does not even come to the point of trying to install any compiled package).

**Comment by Darthyson** [comment]
Same issue for me on Raspi 4 with Debian 12 bookworm.
Last successful build with commit 934b6c1

**Comment by dallmair** [comment]
Fixed with merge of #611
